### PR TITLE
Add option to specify a path or url to raven-js

### PR DESCRIPTION
**Pull request by metalinspired**

### CODE REVIEW

File: config/zend-sentry.global.php.dist [hunk 1]
```diff
@@ -79,6 +79,11 @@ $settings = [
      */
     'ravenjs-version' => '3.27.0',
 
+    /**
+     * If you do not wish to use CDN to load raven-js you can specify path to raven-js to use
```
Comment (markushausammann) [inline config/zend-sentry.global.php.dist]: Please correct the language by using the articles where they’re needed. The CDN, a path. And specify what kind of path you mean, relative to what. Add example.

**Review by markushausammann — CHANGES_REQUESTED** [review]
Please also update the readme documentation of the config options.

**Comment by markushausammann** [comment]
Thanks for this useful PR.

File: config/zend-sentry.global.php.dist (hunk 1 above)
Comment (metalinspired) [reply config/zend-sentry.global.php.dist]: Done
I hope that does it since I've always had problems describing things :)

**Review by markushausammann — APPROVED** [review]

**Comment by markushausammann** [comment]
Thanks again for the PR, I'll prepare a new release as soon as I can.

**Comment by markushausammann** [comment]
@metalinspired since my computer is in repair and I'm depending on a backup infrastructure I'm not really well equipped to create and test a release right now. Can you work with the master for now until I have my computer back?

**Comment by metalinspired** [comment]
@markushausammann no worries :)

**Pull request merged**